lib/std/genasts.nim

```diff
@@ -0,0 +1,71 @@
+import std/macros
```

Comment:
This should be `import macros` as you're already in the `std` sub directory.

Reply:
done, but https://github.com/nim-lang/RFCs/issues/267 would make import syntax non-ambiguous unlike current syntax where it's not clear whether you're importing a module from same package or an external one that has same name